src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +22,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(SEPARATOR))
+                .flatMap(Arrays::stream)
```

Comment:
![image](https://user-images.githubusercontent.com/80981426/210170661-f48bc908-71c1-40d8-b75f-4a7711d932d5.png)
```suggestion
                .flatMap(s -> Arrays.stream(s.split(SEPARATOR)))
```
check similar cases